eng/common/scripts/update-docs-ci.ps1

```diff
@@ -81,7 +81,7 @@ function UpdateParamsJsonPython($pkgs, $ciRepo, $locationInDocRepo){
             install_type = "pypi"
             name=$releasingPkg.PackageId
           }
-          excludePath = @("test*","example*","sample*","doc*")
+          exclude_path = @("test*","example*","sample*","doc*")
```

Comment:
Just curious is this a bug or does python CI make this change recently?

Reply:
Recently.